airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```diff
@@ -0,0 +1,378 @@
+version: 0.65.0
+type: DeclarativeSource
+
+definitions:
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_lever_hiring/schemas/{{ parameters['name'] }}.json"
+
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    client_id: '{{ config[''credentials''][''client_id''] }}'
+    client_secret: '{{ config[''credentials''][''client_secret''] }}'
+    refresh_token: '{{ config[''credentials''][''refresh_token''] }}'
+    refresh_request_body: { }
+    token_refresh_endpoint: '{{ ''https://sandbox-lever.auth0.com/oauth/token'' if config[ ''environment'' ] == ''Sandbox'' else ''https://auth.lever.co/oauth/token'' }}'
+    grant_type: refresh_token
+
+  basic_authenticator:
+    type: BasicHttpAuthenticator
+    username: "{{ config[''credentials''][''api_key''] }}"
+    password: ""
+
+  selective_authenticator:
+    type: SelectiveAuthenticator
+    authenticator_selection_path: [ "credentials", "auth_type" ]
+    authenticators:
+      Client: "#/definitions/oauth_authenticator"
+      Api_Key: "#/definitions/basic_authenticator"
+
+  record_selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - data
+
+  requester:
+    type: HttpRequester
+    url_base: >-
+      {{ "https://api.sandbox.lever.co/v1" if config['environment'] ==
+      "Sandbox" else "https://api.lever.co/v1" }}
+    http_method: GET
+    request_headers: { }
+    request_body_json: { }
+    authenticator:
+      $ref: "#/definitions/selective_authenticator"
+
+  paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: offset
+    page_size_option:
+      inject_into: request_parameter
+      field_name: limit
+      type: RequestOption
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 50
+      cursor_value: '{{ response[''next''] }}'
+      stop_condition: '{{ response[''hasNext''] is false }}'
+
+  opportunities_stream:
+    type: DeclarativeStream
+    $parameters:
+      name: "opportunities"
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/record_selector"
+      requester:
+        $ref: "#/definitions/requester"
+        path: /opportunities
+        request_parameters:
+          include: followers
+          confidentiality: all
```

Comment:
I don't think we should include this parameter. While it is defined as a `base_params` in the Python code, that field is never used.

Reading from the opportunities stream fails when this param is added to the query, but it succeeds when it isn't

> Request to https://api.sandbox.lever.co/v1/opportunities?include=followers&confidentiality=all&limit=50&updated_at_start=1614556800 failed with status code 403 and error message The key you provided does not have access to confidential dat